packages/protocol/migrationsConfig.js

```diff
@@ -69,8 +70,7 @@ const DefaultConfig = {
     spread: 5 / 1000,
     reserveFraction: 1 / 100,
```

Comment:
Must be increased due to frozen reserve cGLD
```suggestion
    reserveFraction: 1 / 20,
```